src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -453,13 +454,13 @@ function diffuse_fraction(
     T::FT,
     P::FT,
     q::FT,
-    SW_IN::FT,
+    SW_d::FT,
```

Comment:
Likewise, at the moment we calculate the fraction of diffuse with an empirical function, but when the model is coupled to the atmosphere, we can read in that fraction from CliMA atmos. FYI.

Reply:
thanks for the reminder!